coriolis/constants.py

```diff
@@ -13,15 +13,20 @@
 TASK_STATUS_CANCELED_FOR_DEBUGGING = "CANCELED_FOR_DEBUGGING"
 TASK_STATUS_ON_ERROR_ONLY = "EXECUTE_ON_ERROR_ONLY"
 
-TASK_TYPE_EXPORT_INSTANCE = "EXPORT_INSTANCE"
-TASK_TYPE_IMPORT_INSTANCE = "IMPORT_INSTANCE"
-TASK_TYPE_FINALIZE_IMPORT_INSTANCE = "FINALIZE_IMPORT_INSTANCE"
-TASK_TYPE_CLEANUP_FAILED_IMPORT_INSTANCE = "CLEANUP_FAILED_IMPORT_INSTANCE"
-
-TASK_TYPE_DEPLOY_DISK_COPY_RESOURCES = "DEPLOY_DISK_COPY_RESOURCES"
-TASK_TYPE_COPY_DISK_DATA = "COPY_DISK_DATA"
-TASK_TYPE_DELETE_DISK_COPY_RESOURCES = "DELETE_DISK_COPY_RESOURCES"
-
+TASK_TYPE_DEPLOY_MIGRATION_SOURCE_RESOURCES = (
+    "DEPLOY_MIGRATION_SOURCE_RESOURCES")
+TASK_TYPE_DEPLOY_MIGRATION_TARGET_RESOURCES = (
+    "DEPLOY_MIGRATION_TARGET_RESOURCES")
+TASK_TYPE_DELETE_MIGRATION_SOURCE_RESOURCES = (
+    "DELETE_MIGRATION_SOURCE_RESOURCES")
+TASK_TYPE_DELETE_MIGRATION_TARGET_RESOURCES = (
+    "DELETE_MIGRATION_TARGET_RESOURCES")
+TASK_TYPE_DEPLOY_INSTANCE_RESOURCES = "DEPLOY_INSTANCE_RESOURCES"
+TASK_TYPE_FINALIZE_INSTANCE_DEPLOYMENT = "FINALIZE_INSTANCE_DEPLOYMENT"
+TASK_TYPE_CLEANUP_FAILED_DEPLOYMENT = "CLEANUP_FAILED_DEPLOYMENT"
```

Comment:
Can you please rename both the variable and the constant to `CLANUP_FAILED_INSTANCE_DEPLOYMENT` to be consistent with the rest?